pages/login.php

```diff
@@ -141,10 +150,12 @@
       <a href="#">Forgot Password?</a>
 
 
-			
+
 	</form>
-  <script src="../js/showpassword.js"></script>  
-<?php include("../include/footer.inc.php") ?>
+</div>
+  <script src="../js/showpassword.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
```

Comment:
why are there **two CDN** links? Aren't they the same?

Reply:
I think there is only 1 cdn link 😅 other one is js for show password

Reply:
I saw another one on top of the file.

Reply:
Wait, I will tag there.